Gopher-Industries/guardian-backend
# Add OTP for Email Verification

## Description

This PR integrates MailerSend as our SMTP relay service to send OTP emails to users (Nurses) for email verification as part of our 2FA security implementation.

### Todos


### How to test

1. `{{base_url}}/api/users/send-pin` to send a pin to an email address while providing the form field `email`.
2. `{{base_url}}/api/users/verify-otp` to verify the pin while providing the form fields `email` and `otp`.

### Screenshots and/or Gifs

<img width="1136" alt="Screenshot 2024-09-23 at 1 10 52 pm" src="https://github.com/user-attachments/assets/60c195e1-886f-4433-a319-2546551e0a96">

### Associated MS Planner Tasks

- [PIN Code Feature Implementation](https://planner.cloud.microsoft/deakin365.onmicrosoft.com/Home/Task/yqy3s-yVgEaGUQUEzJ2VssgADEdT?Type=TaskLink&Channel=Link&CreatedTime=638626582601650000)
